src/App.tsx

```diff
@@ -1,156 +1,101 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { getTodos } from './api/todos';
+import { Login } from './Login';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
 
-const USER_ID = 0;
+const USER_ID = 6846;
```

Comment:
the user_id is hardcoded but it should be dynamic (depending on email)
![image](https://user-images.githubusercontent.com/22824431/229512902-9b7024f4-e181-4ee1-9ec5-5340e52ad4f5.png)